packages/react-router/lib/hooks.tsx

```diff
@@ -1046,14 +1046,14 @@ export function useNavigation() {
 export function useRevalidator() {
```

Comment:
Can we throw existing return types on here to ensure we keep the contract the same?

```suggestion
export function useRevalidator(): {
  revalidate: () => Promise<void>;
  state: DataRouter["state"]["revalidation"];
} {
```